src/main/java/uk/gov/hmcts/reform/sscs/controller/EvidenceManagementController.java

```diff
@@ -38,11 +53,17 @@ public EvidenceManagementController(EvidenceManagementService evidenceManagement
             throw new EvidenceDocumentsMissingException();
         }
 
-        List<MultipartFile> convertedFiles = fileToPdfConversionService.convert(files);
+        try {
+            List<MultipartFile> convertedFiles = fileToPdfConversionService.convert(files);
+            return evidenceManagementService
+                .upload(convertedFiles, DM_STORE_USER_ID)
+                .getEmbedded();
+
+        } catch (FileToPdfConversionException e) {
+            log.error("Error while converting files for evidence upload: " + e.getMessage());
+            throw new FileToPdfConversionException(e.getMessage(), e);
```

Comment:
should be `throw e`, there is no need to wrap the same exception up again

Reply:
agree, changed